### DISCUSSION TITLE
Do we need to be concerned about Notes that appear after running all test cases

### DISCUSSION THREAD

**Discussion by DelusionArun** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#14] [Q&A] [answered]
After running all test, I get-
Note: Some input files use preview language features.
Note: Recompile with -Xlint:preview for details.

Do we need to care about these notes or are these normal? It does not seem to affect compilation nor execution of the program though.

![CheckingNote](https://user-images.githubusercontent.com/72679566/134168638-eaede63c-89c8-4517-b23f-9d692a484424.jpg)

**Comment by Derppening** [comment, marked as answer]
This is normal. This is the compiler's way of telling (or warning) you that you are using preview features, as preview features are subject to change between Java versions.
